docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,337 @@
+
+GTest installation can be performed either following the `GTest installation instructions <https://google.github.io/googletest/>`_, or including the `Gtest repository <https://github.com/google/googletest>`_ in the workspace directory, running:
+
+.. code-block:: bash
+
+    git clone https://github.com/google/googletest ~/fastdds_qosprofman_ws/src/googletest-distribution
+
+.. note::
+
+    If the colcon deployment is followed, and GTest has been installed in the workspace directory, the GTest dependency needs to be included in the *Library* ``colcon.pkg`` file.
+    Please check the :ref:`colcon build procedure <installation_build_source_linux>` for further information.
+
+CLI tool
+^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager CLI* depends on the tool suite *Library* and also on the following package.
+
+Docopt
+""""""
+
+`docopt <http://docopt.org/>`_ is a CLI description language that defines the interface of the command line applications and generates a parser for it automatically.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libdocopt-dev
+
+GUI tool
+^^^^^^^^
+
+.. warning::
+
+    This tool is still in development process.
+
+Documentation tool
+^^^^^^^^^^^^^^^^^^
+
+The *eProsima Fast DDS QoS Profiles Manager Documentation* depends on the tool suite *Library* and also on the following package.
+
+Doxygen
+"""""""
+
+`Doxygen <https://www.doxygen.nl/>`_ is a C++ documentation generator.
+It is required to build the :ref:`Library API reference <fastdds_qos_profiles_manager_lib_api_reference>`.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install doxygen
+
+Python3 virtual environment (recomended)
+""""""""""""""""""""""""""""""""""""""""
+
+It is recommended to install the *Documentation* tool dependencies in a `python3 <https://www.python.org/downloads/>`_ virtual environment.
+That would avoid polluting the user's installation setup.
+Also, the *Documentation* tool has several dependencies described in the ``requirements.txt`` project file.
+
+.. literalinclude:: /requirements.txt
+    :language: text
+
+.. important::
+
+    Once :ref:`the repository is downloaded <installation_download_project_source_linux>`, deploy the project in a python3 virtual environment and install the ``requirements.txt`` dependencies by running:
+
+    .. code-block:: bash
+
+        # Python3 installation
+        sudo apt install python3.10-venv
+
+        # Virtual environment deployment
+        cd <path_to_virtual_environment_directory>
+        python3 -m venv fastdds_qosprofman_venv
+        source fastdds_qosprofman_venv/bin/activate
+
+        # Required dependencies installation
+        pip3 install -r ~/fastdds_qosprofman_ws/src/Fast-DDS-QoS-Profiles-Manager/docs/requirements.txt
+
+    .. note::
+
+        The ``<path_to_virtual_environment_directory>`` could be inside the workspace directory if project is built using CMake.
```

Comment:
This is too much information. I would only leave the notice about colcon and the virtual environment. This note I would think that it makes more sense in the section of installing the virtual environment.